scripts/postBuild.ts

```diff
@@ -0,0 +1,5 @@
+import {promises} from 'fs';

```

Comment:
Please consider 
```ts
import { promises as fs } from 'fs';
```

or

```ts
import * as fs from 'fs/promises';
```

So that we can refer to these method as `fs` rather than `promises`